src/sparse_ir/kernel.py

```diff
@@ -168,6 +176,14 @@ def get_symmetrized(self, sign):
     @property
     def conv_radius(self): return 40 * self.lambda_
 
+    def weight_func(self, statistics: str):
+        """ Return the weight function for given statistics """
+        assert statistics in "FB"
```

Comment:
Please do not use `assert`s when checking for parameter values ... these may be optimized out and do not provide helpful user feedback. 

```suggestion
        if statistics not in "FB":
            raise ValueError("invalid value of statistics argument")
```